generated/ModelicaParser.c:3929:48: error: macro ‘Absyn__CLASS’ requires 10 arguments, but only 9 given

### Description

building v1.25.2 usually succeeds but not today:
```
[ 29%] Building C object OMCompiler/3rdParty/FMIL/CMakeFiles/fmicapi.dir/src/CAPI/src/FMI1/fmi1_capi.c.o
2025-09-26T06:09:17.2691025Z #9 355.4 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/OMCompiler/3rdParty/FMIL/src/CAPI/src/FMI1/fmi1_capi.c: In function ‘fmi1_capi_get_fcn’:
2025-09-26T06:09:17.2693635Z #9 355.4 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/OMCompiler/3rdParty/FMIL/src/CAPI/src/FMI1/fmi1_capi.c:37:9: warning: variable ‘len’ set but not used [-Wunused-but-set-variable]
2025-09-26T06:09:17.2695116Z #9 355.4    37 |     int len;
2025-09-26T06:09:17.2695494Z #9 355.4       |         ^~~
2025-09-26T06:09:17.2864305Z #9 355.5 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/build/OMCompiler/Parser/generated/ModelicaParser.c: In function ‘class_definition’:
2025-09-26T06:09:17.4480332Z #9 355.5 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/build/OMCompiler/Parser/generated/ModelicaParser.c:3929:48: error: macro ‘Absyn__CLASS’ requires 10 arguments, but only 9 given
2025-09-26T06:09:17.4482392Z #9 355.5  3929 |                       PARSER_INFO(retval.start));
2025-09-26T06:09:17.4483159Z #9 355.5       |                                                ^
2025-09-26T06:09:17.4484466Z #9 355.5 In file included from /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/build/OMCompiler/Parser/generated/ModelicaParser.c:887:
2025-09-26T06:09:17.4487153Z #9 355.5 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/OMCompiler/Parser/../Compiler/OpenModelicaBootstrappingHeader.h:1880:9: note: macro ‘Absyn__CLASS’ defined here
2025-09-26T06:09:17.4491249Z #9 355.5  1880 | #define Absyn__CLASS(name,partialPrefix,finalPrefix,encapsulatedPrefix,restriction,body,commentsBeforeClass,commentsBeforeEnd,commentsAfterEnd,info) (mmc_mk_box(11, 3,&Absyn_Class_CLASS__desc,name,partialPrefix,finalPrefix,encapsulatedPrefix,restriction,body,commentsBeforeClass,commentsBeforeEnd,commentsAfterEnd,info))
2025-09-26T06:09:17.4494629Z #9 355.5       |         ^~~~~~~~~~~~
2025-09-26T06:09:17.4497001Z #9 355.6 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/build/OMCompiler/Parser/generated/ModelicaParser.c:3924:35: error: ‘Absyn__CLASS’ undeclared (first use in this function); did you mean ‘Absyn__LESS’?
2025-09-26T06:09:17.4499175Z #9 355.6  3924 |                       retval.ast= Absyn__CLASS(cs.name, mmc_mk_bcon(p), mmc_mk_bcon(final), mmc_mk_bcon(e), ct, cs.ast,
2025-09-26T06:09:17.4501454Z #9 355.6       |                                   ^~~~~~~~~~~~
2025-09-26T06:09:17.4502171Z #9 355.6       |                                   Absyn__LESS
2025-09-26T06:09:17.4503745Z #9 355.6 /home/devel/.cache/aurman/openmodelica-omc/src/OpenModelica/build/OMCompiler/Parser/generated/ModelicaParser.c:3924:35: note: each undeclared identifier is reported only once for each function it appears in
2025-09-26T06:09:17.5497635Z #9 355.8 [ 29%] Building C object OMCompiler/3rdParty/FMIL/CMakeFiles/fmicapi.dir/src/CAPI/src/FMI2/fmi2_capi_cs.c.o
2025-09-26T06:09:17.7019775Z #9 356.0 [ 29%] Building C object OMCompiler/3rdParty/FMIL/CMakeFiles/fmicapi.dir/src/CAPI/src/FMI2/fmi2_capi_me.c.o
2025-09-26T06:09:17.8261525Z #9 356.1 make[2]: *** [OMCompiler/Parser/CMakeFiles/omparse.dir/build.make:226: OMCompiler/Parser/CMakeFiles/omparse.dir/generated/ModelicaParser.c.o] Error 1
2025-09-26T06:09:17.9269913Z #9 356.1 make[1]: *** [CMakeFiles/Makefile2:7640: OMCompiler/Parser/CMakeFiles/omparse.dir/all] Error 2
2025-09-26T06:09:17.9270905Z #9 356.1 make[1]: *** Waiting for unfinished jobs....
2025-09-26T06:09:17.9272023Z #9 356.1 [ 29%] Building CXX object OMCompiler/3rdParty/Ipopt-3.13.4/Ipopt/CMakeFiles/ipopt.dir/__/src/Algorithm/IpExactHessianUpdater.cpp.o
2025-09-26T06:09:17.9273478Z #9 356.2 [ 29%] Building C object OMCompiler/3rdParty/FMIL/CMakeFiles/fmicapi.dir/src/CAPI/src/FMI2/fmi2_capi.c.o
```

### Steps to Reproduce

build v1.25.2 tag with cmake

### Expected Behavior

<!--- A clear and concise description of what you expected to happen. -->

### Screenshots

<!--- If applicable, add screenshots to help explain your problem. -->

### Version and OS

<!--- Please complete the following information. -->

 - OpenModelica Version: 1.25.2
 - OS: archlinux
 - Versions of used Modelica libraries if applicable

### Additional Context

maybe caused by a change in boostrapping sources ?
https://github.com/OpenModelica/OMBootstrapping/pull/9
/cc @adrpo

--- a/OMCompiler/Compiler/boot/CMakeLists.txt
+++ b/OMCompiler/Compiler/boot/CMakeLists.txt
@@ -7,7 +7,7 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/bomc/sources.tar.gz")
 else()
 #download and unpack the sources
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bomc/)
-file(DOWNLOAD https://github.com/OpenModelica/OMBootstrapping/archive/refs/heads/master.tar.gz
+file(DOWNLOAD https://github.com/OpenModelica/OMBootstrapping/archive/04d16f7.tar.gz
      ${CMAKE_CURRENT_SOURCE_DIR}/bomc/sources.tar.gz
      SHOW_PROGRESS
      STATUS status

